src/main/scala/org/broadinstitute/dsde/firecloud/model/Ontology.scala

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.dsde.firecloud.model
+
+
+object Ontology {
+
+  case class TermResource(
+    id: String,
+    ontology: String,
+    usable: Boolean,
+    label: String,
+    definition: String,
+    synonyms: List[String],
+    parents: List[TermParent]
```

Comment:
synonyms and parents are both optional - there are lots of cases where they don't exist. Might make sense to use Options.